stories/exercises/05-customize-theme-tokens.mdx

```diff
@@ -0,0 +1,123 @@
+import { Meta, ColorPalette, ColorItem } from "@storybook/blocks";
+
+# 05. Customize with theme tokens
+
+With content on our landing page, we're going to make sweeping changes using USWDS Design Tokens and theme settings.
+
+You can view all available theme settings by visiting the [USWDS Settings Table](https://designsystem.digital.gov/documentation/settings/#general-settings-2).
+
+Lets start by updating some of our [theme color tokens](https://designsystem.digital.gov/documentation/settings/#color-settings-2). USWDS theme color tokens are designed to be useful and effective for any project size.
+
+1. Select either color palette (Earth tones or Oceanic)  and copy the SCSS theme settings variables.
+
+      ## Colors
+      ### Earth tones
```

Comment:
USWDS theme color tokens are designed to be useful and effective for any project size.

```suggestion
      ## Colors
      
      ### Earth tones
```